Makefile

```diff
@@ -101,6 +102,10 @@ build-staterecoverer:
 build-readtip:
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_READTIP) -v ./tools/readtip
 
+.PHONY: build-timemachine
+build-timemachine:
```

Comment:
do not need this line?

Reply:
add better. I add it  referring to other commands.